shopify_python/google_styleguide.py

```diff
@@ -251,3 +256,48 @@ def __use_cond_expr(self, node):  # type: (astroid.If) -> None
                 else_body_name = node.orelse[0].targets[0].name
                 if if_body_name == else_body_name:
                     self.add_message('cond-expr', node=node)
+
+    def __lambda_func(self, node):  # type: (astroid.Lambda) -> None
+        """Prefer Operator Function to Lambda Functions"""
+
+        if isinstance(node.body, astroid.UnaryOp):
```

Comment:
I think you will false-fail on `f = lambda x: x * 3`.

Please add a test to ensure that  you don't.